src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -2308,6 +2326,106 @@ components:
       required:
         - interests
 
+    UpdateInductionRequest:
+      title: UpdateInductionRequest
+      description: A request to update a Prisoner's Induction.
+      type: object
+      properties:
+        reference:
```

Comment:
Interesting that you added `reference` to this top level request object ... did you have any thoughts about passing it as a path parameter to perhaps make the API a little more RESTful in this respect? 🤔

Reply:
As discussed, I've followed the same approach as we have for `/action-plans` and `/timelines`.

Perhaps the question is not so much about whether we should have `reference` in the path, but more of whether these objects should have a reference at all? Something to think about but I'm not going to change that now :)